core/src/main/java/com/vityuk/ginger/loader/PropertiesLocalizationLoader.java

```diff
@@ -296,8 +297,18 @@ public Double getDouble(String key) {
         @Override
         public List<String> getStringList(String key) {
             String value = get(key);
-            return value == null ? null : Collections.unmodifiableList(
-                    Lists.newArrayList(ARRAY_SPLITTER.split(value)));
+            if(value == null) {
+                return null;
+            }
+            String[] split = value.split(",");
```

Comment:
This is really inefficiently implemented in JDK. I would suggest doing Pattern.compile(",") and reuse it, since that's would it does on every value.split(",") anyway.